config/routes.rb

```diff
@@ -17,25 +17,12 @@
     end
 
   # Special (/ref-based) Loc/Id routes
-  specified_get '/ref/:reference/:repository_id/*locid' => 'ontologies#show',
```

Comment:
Please leave this route in this file, but change the first line to
```ruby
specified_get '/ref/:reference/:repository_id/*locid' => 'ontologies#show',
```

There is a slight difference to the behaviour after this commit:

When requesting text/html of http://ontohub.dev/default/Cat on this commit, the response is a download of the file *without* the file extension.

When requesting text/html of http://ontohub.dev/default/Cat on this commit, the response is a download of the file *with* the file extension.